sdk/core/azure-core-tracing-opentelemetry/pom.xml

```diff
@@ -40,7 +40,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
+      <version>1.3.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
This particular change isn't necessary. Libraries built within the same pipeline _always_ use the **current** version of other libraries built within the same pipeline. Everything built within the core pipeline is already using the latest versions of other things within core. The changes to sdk/eventhubs/azure-messaging-eventhubs/pom.xml are correct, since eventhubs needs the latest, and unreleased, version of core and eventhubs is built within a separate pipeline.